### PR TITLE
ASCN E2E Tests

### PR DESCRIPTION
**Pull request by averyniceday**
- Added E2E Tests
- Requested changes by Niki
	- ASCN Copy # Icon
	- CCF Significant Digits
	- Rename Clonal to Clonality

This is going fail here because this PR is to the feature branch. But previously we were testing by making a PR to master and seeing if they would complete. This is a screenshot of the CircleCI tests.

Screen Shot 2020-06-15 at 4 31 24 PM

This is the same code, squashed and rebased. 

Co-authored-by: Manda Wilson 1458628+mandawilson@users.noreply.github.com
Co-authored-by: Avery Wang 18199796+averyniceday@users.noreply.github.com

## Checks
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

## Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [Giphy CAPTURE](https://giphy.com/apps/giphycapture) or [Peek](https://github.com/phw/peek)

## Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). It can help to figure out who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them either through slack or by assigning them as a reviewer on the PR

### CODE REVIEW

**Comment by averyniceday** [comment]
@zhx828 it's mapping elements to the array of sampleIds coming from the props

File: env/custom.sh [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# set e.g.
```
Comment (onursumer) [inline env/custom.sh:1]: we should not be deleting this one

File: end-to-end-test/local/specs/core/patientview.screenshot.spec.js [hunk 2]
```diff
@@ -6,16 +6,46 @@ var assertScreenShotMatch = require('../../../shared/lib/testUtils')
...
+    CBIOPORTAL_URL + '/patient?studyId=ascn_test_study&caseId=FAKE_P001';
 
 describe('patient view page', function() {
+    describe('mutation table for study with ascn data', () => {
+        beforeEach(() => {
+            goToUrlAndSetLocalStorage(ascnPatientViewUrl);
+            waitForPatientView();
+        });
+
+        it('ascn columns are displayed', () => {
```
Comment (onursumer) [inline end-to-end-test/local/specs/core/patientview.screenshot.spec.js:21]: Convention for the `it` function is that it reads like a proper sentence, so while `it ascn columns are displayed` is not a proper sentence `it displays ascn columns` is. This comment applies most of the `it` functions introduced in this PR.

Comment (averyniceday) [reply end-to-end-test/local/specs/core/patientview.screenshot.spec.js:21]: @onursumer I'll update this today!

File: env/custom.sh (hunk 1 above)
Comment (averyniceday) [reply env/custom.sh:1]: ditto here

**Comment by zhx828** [comment]
> @zhx828 it's mapping elements to the array of sampleIds coming from the props

Does it mean the order has no meaning?

**Comment by averyniceday** [comment]
@zhx828 yeah, the order should just match the order of the samples. Want it to match the `1,2,3,4,5,6..` sample icons in the first column when there's multiple samples per patient.

**Comment by zhx828** [comment]
> @zhx828 yeah, the order should just match the order of the samples. Want it to match the `1,2,3,4,5,6..` sample icons in the first column when there's multiple samples per patient.

I see. It's not so intuitive especially when you have the two columns very far away from each other. But that's not in the scope of this pr.

**Comment by averyniceday** [comment]
@zhx828 yeah, we can always refine down the line 😄  Right now we're running this through Niki and Mark (his group is planning on looking at FACETS data). The original idea was to follow the allele freq column as an example - it's using a bar chart where each bar is a sample.

**Pull request merged**